src/items/class-features/data_broker.json

```
@@ -0,0 +1,178 @@
+    },
+    "descriptors": {
+      "acid": false,
+      "air": false,
+      "calling": false,
+      "chaotic": false,
+      "charm": false,
+      "cold": false,
+      "compulsion": false,
+      "creation": false,
+      "curse": false,
+      "darkness": false,
+      "death": false,
+      "disease": false,
+      "earth": false,
+      "electricity": false,
+      "emotion": false,
+      "evil": false,
+      "fear": false,
+      "fire": false,
+      "force": false,
+      "good": false,
+      "healing": false,
+      "language-dependent": false,
+      "lawful": false,
+      "light": false,
+      "mind-affecting": false,
+      "pain": false,
+      "poison": false,
+      "polymorph": false,
+      "radiation": false,
+      "scrying": false,
+      "sense-dependent": false,
+      "shadow": false,
+      "sonic": false,
+      "summoning": false,
+      "teleportation": false,
+      "water": false
+    },
+    "details": {
+      "category": "classFeature",
+      "combat": false,
+      "specialAbilityType": ""
+    },
+    "duration": {
+      "units": "",
+      "value": null
+    },
+    "formula": "",
+    "isActive": null,
+    "modifiers": [
+      {
+        "_id": "2085727b-8787-47e2-ab0b-2df80f70d6d6",
+        "name": "Specialization",
+        "type": "untyped",
+        "condition": "",
+        "damageGroup": null,
+        "damageTypes": {
+          "acid": false,
+          "bludgeoning": false,
+          "cold": false,
+          "electricity": false,
+          "fire": false,
+          "healing": false,
+          "piercing": false,
+          "slashing": false,
+          "sonic": false
+        },
```

You'll need to do a bit of JSON surgery to these modifiers since they're slightly outdated now:
```suggestion
        "damage": {
            "damageGroup": null,
            "damageTypes": {
            "acid": false,
            "bludgeoning": false,
            "cold": false,
            "electricity": false,
            "fire": false,
            "healing": false,
            "piercing": false,
            "slashing": false,
            "sonic": false
            }
        },
```
You can either do this manually or delete and re-create the modifiers on Foundry, although make sure you're on dev before doing that.